src/Microsoft.Health.Fhir.SqlServer/Features/Storage/SqlServerFhirModel.cs

```diff
@@ -32,14 +31,11 @@ namespace Microsoft.Health.Fhir.SqlServer.Features.Storage
     /// many many times in the database. For more compact storage, we use IDs instead of the strings when referencing these.
     /// Also, because the number of distinct values is small, we can maintain all values in memory and avoid joins when querying.
     /// </summary>
-    public sealed class SqlServerFhirModel : IDisposable
+    public sealed class SqlServerFhirModel : SqlServerModelInitializer
```

Comment:
nit: maybe SqlServerSchemaInitializer since minimally, it needs to initialize the schema and model could be optional?

Reply:
There's already a schema initializer. This class is for initializing the common data (i.e. resource types, systems, etc.). I'm open to calling it something else besides model, but I'm not sure I have a better name.